### DISCUSSION TITLE
How to get TeaVM up and running on an existing LibGDX project

### DISCUSSION THREAD

**Discussion by RikNorakomi** [xpenatan/gdx-teavm#97] [Q&A]
Hello,

TeaVM looks amazing.
I would like to check it out and implement it into an already existing LibGDX kotlin project.
Could you provide us with a brief setup manual with the steps I need to take to add TeaVM to my exisitng project and get the game running on the browser? On the readme page for example?

That would be highly appreciated 👍

**Comment by antzGames** [comment]
Back in February I used https://github.com/xpenatan/gdx-teavm-kotlin-template to get everything working right away.  It is a Kotlin template project.

Hope it still works for you.

**Comment by customautosys** [comment]
Hi I need help here too. Is there any guide on how to create a gdx-teavm subproject?

**Reply by xpenatan** [reply]
@customautosys its best to use liftoff to generate a team module. From there you copy it to your project.

Its best to try testing teavm at the start of your project.  The reason for this is because teavm dont support everything from java/kotlin. I'll soon update kotlin template.
